docs/README-develop-node-utils-APIs.md

```diff
@@ -179,7 +179,9 @@ end
 
 * A minitest should be created to validate the new APIs. Minitests are stored in the tests directory: `tests/`
 
-* Tests may use `config(cmd1, cmd2, ...)` and `@device.cmd("show ...")` to access the CLI directly set up tests and validate expected outcomes. The tests directory contains many examples of how these are used.
+* Tests may use `config(cmd1, cmd2, ...)` to access the CLI directly to set up tests.
+
+* Tests may use `@device.cmd("show ...")` or the helper methods `assert_show_match` and `refute_show_match` to validate expected outcomes in the CLI. The tests directory contains many examples of how these are used.
```

Comment:
Can you add a line to the readme describing @default_show_command and @default_output_pattern?